tests/foreman/ui/test_discoveredhost.py

```diff
@@ -143,6 +145,183 @@ def test_positive_pxe_less_discovery(self):
                 self._assertdiscoveredhost(hostname)
                 self.assertIsNotNone(self.discoveredhosts.search(hostname))
 
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_pxe_less_with_dhcp_semiauto(self):
+        """Discover a host with dhcp via bootable discovery ISO in
+        semi-automated mode.
+
+        @Feature: Foreman Discovery - PXELess
+
+        @Setup: Provisioning should be configured
+
+        @Steps: Boot a host/VM using discovery ISO
+
+        @Assert: Host should be successfully discovered
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_pxe_less_with_dhcp_interactively(self):
+        """Discover a host with dhcp via bootable discovery ISO using
+        interactive TUI mode.
+
+        @Feature: Foreman Discovery - PXELess
+
+        @Setup: Provisioning should be configured
+
+        @Steps: Boot a host/VM using discovery ISO
+
+        @Assert: Host should be successfully discovered
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_pxe_less_without_dhcp_interactively(self):
+        """Discover a host with single NIC on a network without DHCP and PXE
+        using ISO image in interactive TUI interface.
+
+        @Feature: Foreman Discovery
+
+        @Assert: Host should be discovered successfully
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_pxe_less_without_dhcp_semiauto(self):
+        """Discover a host with single NIC on a network without DHCP and PXE
+        using ISO image in semi-automated mode.
+
+        @Feature: Foreman Discovery
+
+        @Assert: Host should be discovered successfully
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_pxe_less_without_dhcp_unattended(self):
+        """Discover a host with single NIC on a network without DHCP and PXE
+        using ISO image in unattended mode.
+
+        @Feature: Foreman Discovery
+
+        @Assert: Host should be discovered successfully
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_discover_pxeless_efi_host_interatively(self):
```

Comment:
should this use `with` for consistency? like

`test_positive_discover_pxe_less_with_efi_host_interatively`

Its okay if the test names are longer :)